### PR TITLE
Get text-gen w/o DS working as a baseline

### PR DESCRIPTION
**Pull request by lekurile**
This PR gets text-generation working in the `examples/generate_text.sh` example working w/o DS inference enabled as a baseline. The `sample_sequence_batch` function has been updated to perform the softmax when calculating `log_probs`, instead of setting it to 1's using `torch.ones_like(...)`. Extra whitespace is also removed.

Currently, the `examples/generate_text.sh` example produces the same nonsensical output irrespective of input context or whether DS inference is enabled or not:
<pre>
Oz jewelry Hod BombCook hiber csivalry Consumptionysics discipline wholesale aversionwic authoritativeduct...
</pre>
This PR gets the example working w/o DS inference, e.g.:
<pre>
Context: Hiking in nature is a

Megatron-LM:  great way to immerse yourself in nature and to get some exercise...
</pre>

**TODO:** DS inference is still giving incorrect output, although the output varies depending on the context:
<pre>
Context: Hiking in nature is a

Megatron-LM:  Over Central Por Over Iz Iz Our Iz Iz Amb Our IB
</pre>

### CODE REVIEW

**Review by RezaYazdaniAminabadi — APPROVED** [review]
All LGTM :)
